### ISSUE TITLE
[Windows][Bug?]: Broken starter

### ISSUE THREAD

**Issue by stevemk42** [solidjs/solid-start#1545]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Current behavior 😯

Hello here.
This may be a silly question, but my installation is broken from the start.
After npm or pnpm (i tried both), hop in the directory and running npm i / pnpm i,
I get a warning about 7 dependencies problem:

`7 deprecated subdependencies found: are-we-there-yet@2.0.0, gauge@3.0.2, glob@7.2.3, glob@8.1.0, inflight@1.0.6, npmlog@5.0.1, rimraf@3.0.2
`
Then when install completed, npm run dev / pnpm run dev return an http 500, with this at localhost:3000:

{
  "statusCode": 500,
  "stack": []
}

and this in my terminal:

```
12:17:19 [vite] Error when evaluating SSR module /@fs/E:/2024/SolidStart/solid-project/node_modules/.pnpm/vinxi@0.3.11_@opentelemetry+api@1.9.0_@types+node@20.14.2_ioredis@5.4.1_terser@5.31.1/node_modules/vinxi/lib/routes.js:
|- Error: Should be provided by compiler
    at eval (E:/2024/SolidStart/solid-project/node_modules/.pnpm/vinxi@0.3.11_@opentelemetry+api@1.9.0_@types+node@20.14.2_ioredis@5.4.1_terser@5.31.1/node_modules/vinxi/lib/routes.js:3:20)
    at instantiateModule (file:///E:/2024/solidstart/solid-project/node_modules/.pnpm/vite@5.3.1_@types+node@20.14.2_terser@5.31.1/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53408:11)

12:17:19 [vite] Error when evaluating SSR module /@fs/E:/2024/SolidStart/solid-project/node_modules/.pnpm/@solidjs+start@1.0.1_rollup@4.18.0_solid-js@1.8.17_vinxi@0.3.11_@opentelemetry+api@1.9.0_@typ_kcf5e2yfvx3itms7hstcfsdd2e/node_modules/@solidjs/start/dist/router/routes.js:       
|- Error: Should be provided by compiler
    at eval (E:/2024/SolidStart/solid-project/node_modules/.pnpm/vinxi@0.3.11_@opentelemetry+api@1.9.0_@types+node@20.14.2_ioredis@5.4.1_terser@5.31.1/node_modules/vinxi/lib/routes.js:3:20)
    at instantiateModule (file:///E:/2024/solidstart/solid-project/node_modules/.pnpm/vite@5.3.1_@types+node@20.14.2_terser@5.31.1/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53408:11)

12:17:19 [vite] Error when evaluating SSR module /@fs/E:/2024/SolidStart/solid-project/node_modules/.pnpm/@solidjs+start@1.0.1_rollup@4.18.0_solid-js@1.8.17_vinxi@0.3.11_@opentelemetry+api@1.9.0_@typ_kcf5e2yfvx3itms7hstcfsdd2e/node_modules/@solidjs/start/dist/router/FileRoutes.js:   
|- Error: Should be provided by compiler
    at eval (E:/2024/SolidStart/solid-project/node_modules/.pnpm/vinxi@0.3.11_@opentelemetry+api@1.9.0_@types+node@20.14.2_ioredis@5.4.1_terser@5.31.1/node_modules/vinxi/lib/routes.js:3:20)
    at instantiateModule (file:///E:/2024/solidstart/solid-project/node_modules/.pnpm/vite@5.3.1_@types+node@20.14.2_terser@5.31.1/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53408:11)

12:17:19 [vite] Error when evaluating SSR module /@fs/E:/2024/SolidStart/solid-project/node_modules/.pnpm/@solidjs+start@1.0.1_rollup@4.18.0_solid-js@1.8.17_vinxi@0.3.11_@opentelemetry+api@1.9.0_@typ_kcf5e2yfvx3itms7hstcfsdd2e/node_modules/@solidjs/start/dist/router/index.jsx:       
|- Error: Should be provided by compiler
    at eval (E:/2024/SolidStart/solid-project/node_modules/.pnpm/vinxi@0.3.11_@opentelemetry+api@1.9.0_@types+node@20.14.2_ioredis@5.4.1_terser@5.31.1/node_modules/vinxi/lib/routes.js:3:20)
    at instantiateModule (file:///E:/2024/solidstart/solid-project/node_modules/.pnpm/vite@5.3.1_@types+node@20.14.2_terser@5.31.1/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53408:11)

12:17:19 [vite] Error when evaluating SSR module /src/app.tsx:
|- Error: Should be provided by compiler
    at eval (E:/2024/SolidStart/solid-project/node_modules/.pnpm/vinxi@0.3.11_@opentelemetry+api@1.9.0_@types+node@20.14.2_ioredis@5.4.1_terser@5.31.1/node_modules/vinxi/lib/routes.js:3:20)
    at instantiateModule (file:///E:/2024/solidstart/solid-project/node_modules/.pnpm/vite@5.3.1_@types+node@20.14.2_terser@5.31.1/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53408:11)

12:17:19 [vite] Error when evaluating SSR module /@fs/E:/2024/SolidStart/solid-project/node_modules/.pnpm/@solidjs+start@1.0.1_rollup@4.18.0_solid-js@1.8.17_vinxi@0.3.11_@opentelemetry+api@1.9.0_@typ_kcf5e2yfvx3itms7hstcfsdd2e/node_modules/@solidjs/start/dist/server/StartServer.jsx: 
|- Error: Should be provided by compiler
    at eval (E:/2024/SolidStart/solid-project/node_modules/.pnpm/vinxi@0.3.11_@opentelemetry+api@1.9.0_@types+node@20.14.2_ioredis@5.4.1_terser@5.31.1/node_modules/vinxi/lib/routes.js:3:20)
    at instantiateModule (file:///E:/2024/solidstart/solid-project/node_modules/.pnpm/vite@5.3.1_@types+node@20.14.2_terser@5.31.1/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53408:11)

12:17:19 [vite] Error when evaluating SSR module /@fs/E:/2024/SolidStart/solid-project/node_modules/.pnpm/@solidjs+start@1.0.1_rollup@4.18.0_solid-js@1.8.17_vinxi@0.3.11_@opentelemetry+api@1.9.0_@typ_kcf5e2yfvx3itms7hstcfsdd2e/node_modules/@solidjs/start/dist/server/index.jsx:       
|- Error: Should be provided by compiler
    at eval (E:/2024/SolidStart/solid-project/node_modules/.pnpm/vinxi@0.3.11_@opentelemetry+api@1.9.0_@types+node@20.14.2_ioredis@5.4.1_terser@5.31.1/node_modules/vinxi/lib/routes.js:3:20)
    at instantiateModule (file:///E:/2024/solidstart/solid-project/node_modules/.pnpm/vite@5.3.1_@types+node@20.14.2_terser@5.31.1/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53408:11)

12:17:19 [vite] Error when evaluating SSR module /src/entry-server.tsx:|- Error: Should be provided by compiler
    at eval (E:/2024/SolidStart/solid-project/node_modules/.pnpm/vinxi@0.3.11_@opentelemetry+api@1.9.0_@types+node@20.14.2_ioredis@5.4.1_terser@5.31.1/node_modules/vinxi/lib/routes.js:3:20)
    at instantiateModule (file:///E:/2024/solidstart/solid-project/node_modules/.pnpm/vite@5.3.1_@types+node@20.14.2_terser@5.31.1/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53408:11)

12:17:19 [vite] Error when evaluating SSR module #vinxi/handler/ssr:
|- Error: Should be provided by compiler
    at eval (E:/2024/SolidStart/solid-project/node_modules/.pnpm/vinxi@0.3.11_@opentelemetry+api@1.9.0_@types+node@20.14.2_ioredis@5.4.1_terser@5.31.1/node_modules/vinxi/lib/routes.js:3:20)
    at instantiateModule (file:///E:/2024/solidstart/solid-project/node_modules/.pnpm/vite@5.3.1_@types+node@20.14.2_terser@5.31.1/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53408:11)

[h3] [unhandled] H3Error: Should be provided by compiler
    at eval (E:/2024/SolidStart/solid-project/node_modules/.pnpm/vinxi@0.3.11_@opentelemetry+api@1.9.0_@types+node@20.14.2_ioredis@5.4.1_terser@5.31.1/node_modules/vinxi/lib/routes.js:3:20)
    at instantiateModule (file:///E:/2024/solidstart/solid-project/node_modules/.pnpm/vite@5.3.1_@types+node@20.14.2_terser@5.31.1/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53408:11) {
  cause: Error: Should be provided by compiler
      at eval (E:/2024/SolidStart/solid-project/node_modules/.pnpm/vinxi@0.3.11_@opentelemetry+api@1.9.0_@types+node@20.14.2_ioredis@5.4.1_terser@5.31.1/node_modules/vinxi/lib/routes.js:3:20)
      at instantiateModule (file:///E:/2024/solidstart/solid-project/node_modules/.pnpm/vite@5.3.1_@types+node@20.14.2_terser@5.31.1/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53408:11),
  statusCode: 500,
  fatal: false,
  unhandled: true,
  statusMessage: undefined,
  data: undefined
}

```

Please help me figure out what this is.

### Steps to reproduce 🕹

Steps:

From Windows, VS Code inner terminal
1. npm create solid / pnpm create solid
2. cd my-app (directory created by naming the new app, example my-app here)
3. npm i / pnpm i
4. npm run dev / pnpm dev / pnpm run dev

### Context 🔦

Windows 10
### Your environment 🌎

**Windows 10**, 👍 
Édition	Windows 10 Family
Version	22H2
Installed  ‎04/13/‎2023
System build	19045.4529
Expérience	Windows Feature Experience Pack 1000.19058.1000.0
Processor	AMD Ryzen 5 2600 Six-Core Processor               3.40 GHz
RAM 	16,0 Go
ID peripheral	5BD23CC2-2561-4A77-869C-3A36E3F752D7
ID product	00326-10122-40373-AA251
System	64-bit operating system, x64 processor

**VS Code** 👍 Version: 1.83.1 (system setup)
Commit: a6606b6ca720bca780c2d3c9d4cc3966ff2eca12
Date: 2023-10-27T12:03:04.063Z
Electron: 25.8.4
ElectronBuildId: 24154031
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT ia32 10.0.19045

**Comment by brenelz** [comment]
It would be beneficial to know which starter your having issues with. Also what operating system are you running on?

**Comment by stevemk42** [comment]
I tried bare, basic, with-auth, with-authjs and with-tailwindcss.
Both with npm and pnpm.
Im using Windows.

Advice came in to try on Mac , which I just tried: Successful, my bare template launched !

Definitely looks like a (my) Windows problem.
Im available to anyone willing to investigate this Windows matter with the starter templates, since my mac is a tiny laptop I cant do serious development on.
Thanks in advance

**Comment by brenelz** [comment]
I wonder if this is always a problem for windows users or if its just your windows setup

**Comment by stevemk42** [comment]
Guess its what it looks like. 
But I have no idea where to start investigate this.
Id like this reporting to help so that other Windows users not having to go through this. 
Kinda sucks.

**Comment by ryansolid** [comment]
I wonder if it is recent. The creator/maintain of the create-solid CLI runs on windows too so I wonder if this is a newer issue as we were testing stuff around the CLI migration. It is possible to be specific to the machine/environment but I don't want to jump to that. Maybe we can get more responses from windows users. I'm going to update the title appropriately.

**Comment by stevemk42** [comment]
Im available for any more information towards investigating this... Im gonna run Solidstart on Mac in the meantime. Honestly, I think its recent, I tried just 1 week ago... I reported this after spending 2/3 days on npm clean cache and other stack overflow tricks.

**Comment by oddpxl** [comment]
Similar problems...   MacOS Sonoma 14.5

pnpm create solid

┌  
 Create-Solid v0.5.12
│
◇  Project Name
│  TestWeb
│
◇  Is this a Solid-Start project?
│  Yes
│
◇  Which template would you like to use?
│  with-tailwindcss
│
◇  Use Typescript?
│  Yes
│
◇  Project successfully created! 🎉
│
◇  To get started, run: ─╮
│                        │
│  cd TestWeb             │
│  pnpm install          │
│  pnpm dev              │
│                        │
├────────────────────────╯

cd LizWeb
pnpm install

**WARN  9 deprecated subdependencies found: are-we-there-yet@2.0.0, gauge@3.0.2, glob@7.2.3, glob@8.1.0, inflight@1.0.6, npmlog@5.0.1, rimraf@3.0.2, shikiji-core@0.9.19, shikiji@0.9.19**

Packages: +520
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Progress: resolved 590, reused 520, downloaded 0, added 520, done

dependencies:
+ @solidjs/router 0.14.1
+ @solidjs/start 1.0.6
+ autoprefixer 10.4.20
+ postcss 8.4.41
+ solid-js 1.8.19
+ tailwindcss 3.4.8
+ vinxi 0.4.1

Done in 5.1s

**Comment by curiouscoder13** [comment]
Windows 10 user here, I have tried base template and it works, no problem, the issue I have is with 'with-authjs' template. here is the exec:

npm init solid@latest

> npx
> create-solid

┌
 Create-Solid v0.5.12
│
◇  Project Name
│  auth-app-solid
│
◇  Is this a Solid-Start project?
│  Yes
│
◇  Which template would you like to use?
│  with-authjs
│
◇  Use Typescript?
│  No
│
◇  Project successfully created! 🎉
│
◇  To get started, run: ─╮
│                        │
│  cd auth-app-solid     │
│  npm install           │
│  npm run dev           │
│                        │
├────────────────────────╯

cd auth-app-solid

npm install

npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: example-with-authjs@undefined
npm error Found: @solidjs/router@0.14.3
npm error node_modules/@solidjs/router
npm error   @solidjs/router@"^0.14.1" from the root project
npm error
npm error Could not resolve dependency:
npm error peer @solidjs/router@"^0.13.1" from @solid-mediakit/auth@2.1.2
npm error node_modules/@solid-mediakit/auth
npm error   @solid-mediakit/auth@"^2.0.11" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error
npm error
npm error For a full report see:
npm error C:\Users\deans\AppData\Local\npm-cache\_logs\2024-08-30T01_50_36_683Z-eresolve-report.txt
npm error A complete log of this run can be found in: C:\Users\deans\AppData\Local\npm-cache\_logs\2024-08-30T01_50_36_683Z-debug-0.log

**2024-08-30T01_50_36_683Z-eresolve-report.txt** content:
'Log files:
C:\Users\deans\AppData\Local\npm-cache\_logs\2024-08-30T01_50_36_683Z-debug-0.log

# npm resolution error report

While resolving: example-with-authjs@undefined
Found: @solidjs/router@0.14.3
node_modules/@solidjs/router
  @solidjs/router@"^0.14.1" from the root project

Could not resolve dependency:
peer @solidjs/router@"^0.13.1" from @solid-mediakit/auth@2.1.2
node_modules/@solid-mediakit/auth
  @solid-mediakit/auth@"^2.0.11" from the root project

Fix the upstream dependency conflict, or retry
this command with --force or --legacy-peer-deps
to accept an incorrect (and potentially broken) dependency resolution.

**2024-08-30T01_50_36_683Z-debug-0.log** content:
0 verbose cli C:\Program Files\nodejs\node.exe C:\Users\deans\AppData\Roaming\npm\node_modules\npm\bin\npm-cli.js
1 info using npm@10.8.2
2 info using node@v20.14.0
3 silly config load:file:C:\Users\deans\AppData\Roaming\npm\node_modules\npm\npmrc
4 silly config load:file:F:\Mac Oidz\Website Building Projects\__0__FullStack\fullstack-test-project\auth-app-solid\.npmrc
5 silly config load:file:C:\Users\deans\.npmrc
6 silly config load:file:C:\Users\deans\AppData\Roaming\npm\etc\npmrc
7 verbose title npm install
8 verbose argv "install"
9 verbose logfile logs-max:10 dir:C:\Users\deans\AppData\Local\npm-cache\_logs\2024-08-30T01_50_36_683Z-
10 verbose logfile C:\Users\deans\AppData\Local\npm-cache\_logs\2024-08-30T01_50_36_683Z-debug-0.log
11 silly packumentCache heap:4345298944 maxSize:1086324736 maxEntrySize:543162368
12 silly logfile start cleaning logs, removing 1 files
13 silly logfile done cleaning log files
14 silly idealTree buildDeps
15 silly fetch manifest @auth/core@^0.29.0
16 silly packumentCache full:https://registry.npmjs.org/@auth%2fcore cache-miss
17 http fetch GET 200 https://registry.npmjs.org/@auth%2fcore 1324ms (cache miss)
18 silly packumentCache full:https://registry.npmjs.org/@auth%2fcore set size:undefined disposed:false
19 silly fetch manifest @simplewebauthn/browser@^9.0.1
20 silly packumentCache full:https://registry.npmjs.org/@simplewebauthn%2fbrowser cache-miss
21 http fetch GET 200 https://registry.npmjs.org/@simplewebauthn%2fbrowser 605ms (cache miss)
22 silly packumentCache full:https://registry.npmjs.org/@simplewebauthn%2fbrowser set size:undefined disposed:false
23 silly fetch manifest @simplewebauthn/server@^9.0.2
24 silly packumentCache full:https://registry.npmjs.org/@simplewebauthn%2fserver cache-miss
25 http fetch GET 200 https://registry.npmjs.org/@simplewebauthn%2fserver 1042ms (cache miss)
26 silly packumentCache full:https://registry.npmjs.org/@simplewebauthn%2fserver set size:undefined disposed:false
27 silly fetch manifest nodemailer@^6.8.0
28 silly packumentCache full:https://registry.npmjs.org/nodemailer cache-miss
29 http fetch GET 200 https://registry.npmjs.org/nodemailer 1066ms (cache miss)
30 silly packumentCache full:https://registry.npmjs.org/nodemailer set size:undefined disposed:false
31 silly fetch manifest @babel/core@7.24.4
32 silly packumentCache full:https://registry.npmjs.org/@babel%2fcore cache-miss
33 http fetch GET 200 https://registry.npmjs.org/@babel%2fcore 317ms (cache revalidated)
34 silly packumentCache full:https://registry.npmjs.org/@babel%2fcore set size:657010 disposed:false
35 silly fetch manifest @solid-mediakit/auth@^2.0.11
36 silly packumentCache full:https://registry.npmjs.org/@solid-mediakit%2fauth cache-miss
37 http fetch GET 200 https://registry.npmjs.org/@solid-mediakit%2fauth 2288ms (cache miss)
38 silly packumentCache full:https://registry.npmjs.org/@solid-mediakit%2fauth set size:undefined disposed:false
39 silly fetch manifest @solidjs/meta@^0.29.4
40 silly packumentCache full:https://registry.npmjs.org/@solidjs%2fmeta cache-miss
41 http fetch GET 200 https://registry.npmjs.org/@solidjs%2fmeta 1806ms (cache revalidated)
42 silly packumentCache full:https://registry.npmjs.org/@solidjs%2fmeta set size:36043 disposed:false
43 silly fetch manifest solid-js@^1.8.18
44 silly packumentCache full:https://registry.npmjs.org/solid-js cache-miss
45 http fetch GET 200 https://registry.npmjs.org/solid-js 204ms (cache revalidated)
46 silly packumentCache full:https://registry.npmjs.org/solid-js set size:1960202 disposed:false
47 silly fetch manifest @solidjs/router@^0.14.1
48 silly packumentCache full:https://registry.npmjs.org/@solidjs%2frouter cache-miss
49 http fetch GET 200 https://registry.npmjs.org/@solidjs%2frouter 1769ms (cache revalidated)
50 silly packumentCache full:https://registry.npmjs.org/@solidjs%2frouter set size:195774 disposed:false
51 silly fetch manifest @solidjs/router@^0.13.1
52 silly packumentCache full:https://registry.npmjs.org/@solidjs%2frouter cache-hit
53 verbose stack Error: unable to resolve dependency tree
53 verbose stack     at #failPeerConflict (C:\Users\deans\AppData\Roaming\npm\node_modules\npm\node_modules\@npmcli\arborist\lib\arborist\build-ideal-tree.js:1360:25)
53 verbose stack     at #loadPeerSet (C:\Users\deans\AppData\Roaming\npm\node_modules\npm\node_modules\@npmcli\arborist\lib\arborist\build-ideal-tree.js:1326:33)
53 verbose stack     at async #buildDepStep (C:\Users\deans\AppData\Roaming\npm\node_modules\npm\node_modules\@npmcli\arborist\lib\arborist\build-ideal-tree.js:901:11)
53 verbose stack     at async Arborist.buildIdealTree (C:\Users\deans\AppData\Roaming\npm\node_modules\npm\node_modules\@npmcli\arborist\lib\arborist\build-ideal-tree.js:181:7)
53 verbose stack     at async Promise.all (index 1)
53 verbose stack     at async Arborist.reify (C:\Users\deans\AppData\Roaming\npm\node_modules\npm\node_modules\@npmcli\arborist\lib\arborist\reify.js:131:5)
53 verbose stack     at async Install.exec (C:\Users\deans\AppData\Roaming\npm\node_modules\npm\lib\commands\install.js:150:5)
53 verbose stack     at async Npm.exec (C:\Users\deans\AppData\Roaming\npm\node_modules\npm\lib\npm.js:207:9)
53 verbose stack     at async module.exports (C:\Users\deans\AppData\Roaming\npm\node_modules\npm\lib\cli\entry.js:74:5)
54 error code ERESOLVE
55 error ERESOLVE unable to resolve dependency tree
56 error
57 error While resolving: example-with-authjs@undefined
57 error Found: @solidjs/router@0.14.3[2m[22m
57 error [2mnode_modules/@solidjs/router[22m
57 error   @solidjs/router@"^0.14.1" from the root project
57 error
57 error Could not resolve dependency:
57 error [95mpeer[39m @solidjs/router@"^0.13.1" from @solid-mediakit/auth@2.1.2[2m[22m
57 error [2mnode_modules/@solid-mediakit/auth[22m
57 error   @solid-mediakit/auth@"^2.0.11" from the root project
57 error
57 error Fix the upstream dependency conflict, or retry
57 error this command with --force or --legacy-peer-deps
57 error to accept an incorrect (and potentially broken) dependency resolution.
58 error
58 error
58 error For a full report see:
58 error C:\Users\deans\AppData\Local\npm-cache\_logs\2024-08-30T01_50_36_683Z-eresolve-report.txt
59 silly unfinished npm timer reify 1724982637532
60 silly unfinished npm timer reify:loadTrees 1724982637541
61 silly unfinished npm timer idealTree:buildDeps 1724982637564
62 silly unfinished npm timer idealTree:#root 1724982637565
63 verbose cwd F:\Mac Oidz\Website Building Projects\__0__FullStack\fullstack-test-project\auth-app-solid
64 verbose os Windows_NT 10.0.19045
65 verbose node v20.14.0
66 verbose npm  v10.8.2
67 verbose exit 1
68 verbose code 1
69 error A complete log of this run can be found in: C:\Users\deans\AppData\Local\npm-cache\_logs\2024-08-30T01_50_36_683Z-debug-0.log

**Comment by stevemk42** [comment]
Hello here. 
I tried again, even if Im pretty advanced on the Mac version I started sometime ago. 
I'll restart on Windows, seems to work now.
I'm grateful for whatever the developers did.
Thanks @ryansolid 

I'll close this thread with this comment.

Exec (lines keyed in are in bold):

**npx create-solid**

Need to install the following packages:
create-solid@0.5.12
Ok to proceed? (y) y

┌  
 Create-Solid v0.5.12
│
◇  Project Name
│  uai-win
│
◇  Is this a Solid-Start project?
│  Yes
│
◇  Which template would you like to use?
│  with-tailwindcss
│
◇  Use Typescript?
│
◇  Project successfully created! 🎉
│
◇  To get started, run: ─╮
│                        │
│  cd uai-win            │
│  npm install           │
│  npm run dev           │
│                        │
├────────────────────────╯

**cd uai-win
npm install**

npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm warn deprecated npmlog@5.0.1: This package is no longer supported.
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@8.1.0: Glob versions prior to v9 are no longer supported
npm warn deprecated are-we-there-yet@2.0.0: This package is no longer supported.
npm warn deprecated gauge@3.0.2: This package is no longer supported.
npm warn deprecated shikiji-core@0.9.19: Shikiji is merged back to Shiki v1.0, please migrate over to get the latest updates
npm warn deprecated shikiji@0.9.19: Shikiji is merged back to Shiki v1.0, please migrate over to get the latest updates

added 568 packages, and audited 571 packages in 30s

109 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

**npm run dev**

> dev      
> vinxi dev

vinxi v0.4.3
vinxi starting dev server

  ➜ Local:    http://localhost:3000/
  ➜ Network:  use --host to expose  

10:43:52 [vite] page reload vinxi/routes

Rolling !
